cluster/gce/util.sh

```diff
@@ -38,27 +38,36 @@ else
 fi
 
 if [[ "${MASTER_OS_DISTRIBUTION}" == "gci" ]]; then
-  # If the master image is not set, we use the latest GCI image.
-  # Otherwise, we respect whatever is set by the user.
-  MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-${GCI_VERSION}}
-  MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-google-containers}
+    DEFAULT_GCI_PROJECT=google-containers
```

Comment:
I can change the defaults here too. Honestly I don't think it matters. All these hacks would hopefully disappear soon.
At this point, I want this PR to go in and then I can clean things up.